main.go

```diff
@@ -23,6 +23,7 @@ var (
 	prom           = flag.Bool("prometheus", false, "Stats for prometheus monitoring system")
 	promHost       = flag.String("prometheus-host", "127.0.0.1", "IP to bind Prometheus service to")
 	promPort       = flag.Int32("prometheus-port", 8090, "Prometheus port")
+	udp             = flag.Bool("udp", false, "Stats for udp monitoring system")
```

Comment:
Why CLI option given that there is already udp in .json file ?

Reply:
Thanks so much for advice. Please find the changes in commit [remove udp CLI option](https://github.com/Juniper/jtimon/pull/7/commits/6c4a483710f728af3f3737ef4077a4677d4bcd27).